unison-cli/src/Unison/Codebase/Editor/HandleInput/Update2.hs

```diff
@@ -184,10 +187,9 @@ makeParsingEnv path names = do
       }
 
 -- save definitions and namespace
-saveTuf :: (Name -> Either Output (Maybe [Name])) -> TypecheckedUnisonFile Symbol Ann -> Cli ()
-saveTuf getConstructors tuf = do
+saveTuf :: (Name -> Either Output (Maybe [Name])) -> TypecheckedUnisonFile Symbol Ann -> Path.Absolute -> Cli ()
+saveTuf getConstructors tuf currentPath = do
```

Comment:
This is now the `update-in` path right? I see it's used as the `stepAt` path below, what happens if there are propagations above that path, but still within the project?

Reply:
The intention is that there wouldn't be any propagations above that path (equivalent to using `cd` then `update` today)

Reply:
I see from the next example why this could be confusing. It's not meant to be the better thing yet, simply because we haven't taken the time ti implement that. Maybe I should rename this to `cd.update`?

Reply:
or `deprecated.upddate-in`